src/gui/src/displayControls.cpp

```diff
@@ -1452,6 +1452,9 @@ void DisplayControls::techInit()
   int metal = 0;
   int via = 0;
 
+  // ensure if random colors are used they are consistent
+  srand(1);
```

Comment:
This is a global change and I fear that other code may be affected.  I don't want to have cases where you get different results with and without the gui.  Please switch to a random number generator with a local seed (see https://en.cppreference.com/w/cpp/numeric/random).

Reply:
switch to a local random